### DISCUSSION TITLE
Does O3DE have the capability to support pixel streaming?

### DISCUSSION THREAD

**Discussion by 420943103** [o3de/o3de#18232] [Help Wanted]
Does O3DE support a pixel streaming feature similar to Unreal Engine, pushing the rendered content in the form of pixel streams to the front-end browser for display and enabling interactive operations? Is it feasible to develop a similar feature in O3DE based on the implementation code of the pixel streaming plugin of Unreal Engine?

**Comment by guillaume-haerinck** [comment]
There is a WebGPU support being developed so it would run the app in the browser, not as a video stream but as being actually rendered https://github.com/o3de/o3de/pull/18295

**Reply by 420943103** [reply]
Thank you very much for your reply, but I want to render the scene by using the GPU resources of the server, not the local GPU resources, so pixel streaming is more suitable.
